src/stories/Examples/LazyImageLoading/LazyImageLoading.stories.ts

```diff
@@ -0,0 +1,21 @@
+import LazyImageLoading from "./LazyImageLoading.svelte";
+import BlurhashRendererDecorator from "../../decorators/BlurhashRendererDecorator.svelte";
+import type { Meta, StoryObj } from "@storybook/svelte";
+
+const meta = {
```

Comment:
It could be helpful to add a brief [description](https://storybook.js.org/docs/react/api/doc-block-description#writing-descriptions) via the `parameters.docs.description.component` property

Reply:
I've added a full document explaining how to use the example.